actions/gradle-plugin-setup/action.yaml

```diff
@@ -21,12 +21,12 @@ runs:
         fetch-depth: 0
         submodules: ${{ inputs.checkout-submodules }}
     - name: Set up JDK ${{ inputs.java-version }}
-      uses: actions/setup-java@v3
+      uses: actions/setup-java@v4
       with:
         distribution: temurin
         java-version: ${{ inputs.java-version }}
     - name: Setup Gradle
-      uses: gradle/gradle-build-action@v2
+      uses: gradle/actions/setup-gradle@v3
```

Comment:
v4 of `gradle/actions/setup-gradle` is available, please update to that latest major release.